pallets/funding/src/lib.rs

```diff
@@ -554,6 +560,9 @@ pub mod pallet {
 		ValueQuery,
 	>;
 
+	#[pallet::storage]
+	pub type ProjectSettlementQueue<T: Config> = StorageValue<_, BoundedVec<ProjectId, T::MaxProjectsToSettle>, ValueQuery>;
```

Comment:
Maybe use a `WeakBoundedVec` so even if the queue is full, you can still append stuff. Because what would you do if at funding end the appending fails?